web/styles/popovers.css

```diff
@@ -1469,17 +1440,26 @@ ul.navbar-dropdown-menu-outer-list {
     }
 }
 
-.navbar-dropdown-hotkey-hint {
-    color: var(--color-hotkey-hint);
-    text-align: center;
-    font-size: 14px;
-    font-style: normal;
-    font-weight: 500;
-    line-height: 14px;
-    padding: 2px 4px;
-    border-radius: 3px;
-    border: 1px solid var(--color-hotkey-hint);
+.popover-menu-hotkey-hints {
+    display: flex;
+    gap: 4px;
     margin-left: auto;
+    padding-left: 5px;
+
+    > .popover-menu-hotkey-hint {
+        box-sizing: border-box;
+        display: inline-block;
```

Comment:
Why is it necessary to set `inline-block` here? As a child of `.popover-menu-hotkey-hints`, any `.popover-menu-hotkey-hint` should display as a flex item, rendering its `display` value moot.

Reply:
I had put it there to hint towards the fact that the `width` related do apply to the `flex-child` elements. The context for this was that I came across 634a771a37ad475068c1becd29b3dc5c4c2490a4, where this nuance was overlooked (due to which the tooltip hotkeys are no longer squared to a minimum).

Reply:
Got it. Yes, once something is a flex item, the `width` or `flex-basis` values apply.